packages/analytics-js-integrations/src/integrations/Klaviyo/browser.js

```diff
@@ -119,8 +120,9 @@ class Klaviyo {
       $country: country,
       $organization: get(message, 'context.traits.organization'),
       $title: get(message, 'context.traits.title'),
-      $region: get(message, 'context.traits.region'),
-      $zip: get(message, 'context.traits.zip'),
+      $region: get(message, 'context.traits.region') || state,
```

Comment:
let's a add small test case where the newly added code is covered

Reply:
Klaviyo don't have any testcases at all.

Reply:
@Gauravudia It would be better if we can add the test cases in that cases as this is what we are starting doing to cover for integration with no test cases

Reply:
Will raise a separate PR covering this.